src/commands/commit/api.ts

```diff
@@ -0,0 +1,41 @@
+import {AxiosPromise, AxiosRequestConfig, AxiosResponse} from 'axios'
+import FormData from 'form-data'
+import {Writable} from 'stream'
+
+import {getRequestBuilder} from '../../helpers/utils'
+
+import {Payload} from './interfaces'
+import {renderUpload} from './renderer'
+
+// Dependency follows-redirects sets a default maxBodyLength of 10 MB https://github.com/follow-redirects/follow-redirects/blob/b774a77e582b97174813b3eaeb86931becba69db/index.js#L391
+// We don't want any hard limit enforced by the CLI, the backend will enforce a max size by returning 413 errors.
+const maxBodyLength = Infinity
+
+export const uploadRepository = (request: (args: AxiosRequestConfig) => AxiosPromise<AxiosResponse>) => async (
```

Comment:
It does not seem to be imported in other files:
```suggestion
const uploadRepository = (request: (args: AxiosRequestConfig) => AxiosPromise<AxiosResponse>) => async (
```